PrettyConsoleHelper/InputHelper.cs

```diff
@@ -78,6 +78,35 @@ public int GetIntInput(string message = "Enter a whole number", int maxValue = i
             }
         }
 
+        /// <summary>
+        /// Loops until the user has succesfully entered a datetime
+        /// </summary>
+        /// <param name="message"></param>
+        /// <param name="maxDateTime"></param>
+        /// <param name="minDateTime"></param>
+        /// <returns></returns>
+        public DateTime GetDateTime(string message = "Enter a date", DateTime? minDateTime = null, DateTime? maxDateTime = null)
```

Comment:
you can directly assign datetime.minvalue and datetime.maxvalue instead of using null checks

Reply:
I tried but they have to be compile time constant